crates/cli-support/src/wit/mod.rs

```diff
@@ -1513,7 +1520,77 @@ fn verify_schema_matches<'a>(data: &'a [u8]) -> Result<Option<&'a str>, Error> {
 fn concatenate_comments(comments: &[&str]) -> String {
     comments
         .iter()
-        .map(|s| s.trim_matches('"'))
+        .map(|&s| try_unescape(s).unwrap_or_else(|| s.to_owned()))
         .collect::<Vec<_>>()
         .join("\n")
 }
+
+// Unescapes a quoted string. char::escape_debug() was used to escape the text.
+fn try_unescape(s: &str) -> Option<String> {
```

Comment:
I'm personally a bit confused on why we need a method like this, can you help me understand where this is coming from? Where do we acquire an escaped string from, and would it be possible to acquire an unescaped string instead?

Reply:
https://github.com/rustwasm/wasm-bindgen/blob/b9f78aba5765aceac2e4cc3d67ba95a312f4c1cc/crates/macro-support/src/parser.rs#L1320-L1324

That's the code that gets a quoted + escaped string from `proc_macro2`. I couldn't find an unescape function anywhere. If you know of one, I'll replace this func with that one.

Reply:
Can the un-quoting/etc happen around there? I don't think it should be happening in the CLI but rather at the source here if possible.

Reply:
It's been moved now. The old code used to unquote the string, but it's also been moved. Let me know if that's considered a breaking change and I'll restore it.